### PR TITLE
Delete News Update : UI Fixes

### PR DESCRIPTION
**Pull request by kumarpalsinh25**
- fixes, #1320 
- fixes, https://github.com/acterglobal/a3-meta/issues/93

Reference Video:
https://github.com/acterglobal/a3/assets/51526480/2094f7f3-28b0-4673-abdc-48d2036a525e

### CODE REVIEW

File: app/lib/features/news/widgets/news_side_bar.dart [hunk 1]
```diff
@@ -197,7 +197,10 @@ class ActionBox extends ConsumerWidget {
             builder: (context) => RedactContentWidget(
               title: 'Remove this post',
               eventId: news.eventId().toString(),
-              onSuccess: () => ref.invalidate(newsListProvider),
+              onSuccess: () {
+                context.pop();
+                ref.invalidate(newsListProvider);
```
Comment (gtalha07) [inline app/lib/features/news/widgets/news_side_bar.dart:202]: not sure if this indeed works everytime...is it possible to include integration test for deleting update? That'll make us somewhat confident!

**Review by gtalha07 — APPROVED** [review]
Looks good!

File: app/lib/features/news/widgets/news_side_bar.dart (hunk 1 above)
Comment (kumarpalsinh25) [reply app/lib/features/news/widgets/news_side_bar.dart:202]: News reduction bugs was already fixed on past [PR-Live redaction support for spaces.](https://github.com/acterglobal/a3/pull/1448).

In this PR I have just fixed minor UI glitches with  `context.pop();` to close deletion menu after news post deleted successfully.

And yes, Surely I will add end-to-end integration test for News Post Delete in separate PR. :)

Comment (kumarpalsinh25) [reply app/lib/features/news/widgets/news_side_bar.dart:202]: > is it possible to include integration test for deleting update? That'll make us somewhat confident!

Done. Added it here. :)
https://github.com/acterglobal/a3/pull/1463

**Pull request merged**